docs/schema_plugins.md

```diff
@@ -1 +1,234 @@
 # Extending NOMAD-Simulations to support custom methods and outputs
+
+As you develop your parser, you may find that the `nomad-simulations` package does not include some relevant quantities for your particular use case. You can easily extend upon the schema by adding your own custom schema under the `schema_packages/` directory in your parser plugin. For this, you should utilize and build upon `nomad-simulations` for consistency and future compatibility or integration. The following page demonstrates the basics of how to implement your own schema in practice.
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/schema_plugins/NOMAD_Simulations-extending-the-schema.png" alt="Schema extension decision tree." width="80%" title="Click to zoom in">
+    </label>
+</div>
+
+Within your parser plugin project, under `schema_packages/`, create a python file for your schema, e.g., `<parser_name>_schema.py`.
+
+Add the following imports to this file:
+
+```python
+import nomad_simulations
+import numpy as np
+from nomad.datamodel.data import ArchiveSection
+from nomad.metainfo import Context, MEnum, Quantity, Section, SectionProxy, SubSection
+```
+
+- `nomad_simulations`:
+
+- `numpy`:
+
+- `ArchiveSection`:
+
+- `SubSection`:
```

Comment:
What are these for? I think all of these concepts will already have been introduced by now.